apps/dashboard/app/models/transfer.rb

```diff
@@ -7,11 +7,16 @@ class Transfer
   attr_accessor :id, :status, :created_at, :started_at, :completed_at
   attr_writer :percent
 
-  validates_each :files do |record, attr, value|
+  validates_each :files do |record, attr, files|
     if record.action == 'mv' || record.action == 'cp'
-      conflicts = value.values.select {|f| File.exist?(f) }
+      conflicts = files.values.select {|f| File.exist?(f) }
       record.errors.add :files, "these files already exist: #{conflicts.join(', ')}" if conflicts.present?
```

Comment:
This is not really a part of this change, I'm just noticing it now. We're not allowing moves or copies if the destination already exists? `cp` and `mv` allow for this just fine, is there a reason we're differing from their behaviour?

Reply:
Preexisting functionality. We can remove this limitation if we enhance the user interface to provide warning to the user. Typically if a move or copy would end up overwriting existing files and you wanted to do that you would first be presented with a warning and an option to proceed. That will just take more time to get right.

Reply:
Note I think you could open an issue about adding supporting for that.